app/services/data_migrations/delete_uuidless_courses.rb

```diff
@@ -0,0 +1,17 @@
+module DataMigrations
```

Comment:
I think we're missing the data migration class name being added to: https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/lib/tasks/data.rake#L3